beacon_node/http_api/src/lib.rs

```diff
@@ -4652,8 +4652,10 @@ pub fn serve<T: BeaconChainTypes>(
                         post_beacon_blocks_ssz
                             .uor(post_beacon_blocks_v2_ssz)
                             .uor(post_beacon_blinded_blocks_ssz)
-                            .uor(post_beacon_blinded_blocks_v2_ssz),
+                            .uor(post_beacon_blinded_blocks_v2_ssz)
+                            .boxed(),
```

Comment:
this one is redundant, I'm pretty sure, because the result comes straight out of a `uor`:

https://github.com/sigp/lighthouse/blob/441fc1691b69f9edc4bbdc6665f3efab16265c9b/common/warp_utils/src/uor.rs#L16